app/server/logging/jaeger/src/main/java/io/syndesis/server/logging/jaeger/JaegerActivityTrackingConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.server.logging.jaeger;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Configured in spring.factories so that this configuration is automatically picked
+ * up when included in the classpath.
+ */
+@Configuration
+@ComponentScan
+@ConditionalOnProperty(value = "features.jaeger-activity-tracing.enabled", havingValue = "true")
+public class JaegerActivityTrackingConfiguration {
```

Comment:
Needed? I thought we did `@ComponentScan` regardless and it doesn't have any `@Configuration` beans here.